behave/helper/constants.py

```diff
@@ -189,3 +190,5 @@
 CLA_CONTACT_US_USER_PERSON_TO_CALL = "Elena Fisher"
 
 MINIMUM_SLEEP_SECONDS = 2
+
+FALA_HOMEPAGE_HEADER = "Find a legal aid adviser or family mediator"
```

Comment:
Super tiny comment. Can you rename this to FALA_HEADER, as it's the same String on both pages, not just the homepage.

Reply:
Don't forget to reflect the change in name where it referenced in the tests.